src/components/AssetBreadcrumb/stories/basic.story.mdx

```diff
@@ -10,7 +10,7 @@ This component renders breadcrumb for assets chain, that represents path from ro
 
 **NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
 
-## Basic usage:
+### Basic usage:
```

Comment:
We have different headers for `Basic usage` section across whole documentation now. Pls, update all files to match criteria  (`###`)